test/test_manylinuxXXXX_only.py

```diff
@@ -44,9 +45,12 @@
     "manylinux_image", ["manylinux1", "manylinux2010", "manylinux2014", "manylinux_2_24"]
 )
 def test(manylinux_image, tmp_path):
+    pm = platform.machine()
     if utils.platform != "linux":
         pytest.skip("the docker test is only relevant to the linux build")
-    elif platform.machine() not in ["x86_64", "i686"]:
+    elif pm == "ppc64le" and "TRAVIS" in os.environ:
+        pytest.skip("this test requires too much disk space for this runner")
```

Comment:
Alternatively, we could always add this with `-k "not manylinuxXXXX_only` in the actual invocation of pytest on that specific CI (as it feels slightly weird to have CI-specific things in the tests). But this also works perfectly for me, and is probably easier.

Reply:
OK, seems @joerick had the same idea ;-)

Reply:
I did not think of this. This felt natural (maybe it shouldn't) and I think we are more likely to bump into this code now & then rather than just forget we added a specific option in the CI configuration.
I can still try the `PYTEST_ADDOPTS='-k "not test_manylinuxXXXX_only"'`  option, my previous feeling is not a strong one.

Reply:
Also true. As you prefer; it wasn't a very strong, need-to-have suggestion either!

Reply:
> OK, seems @joerick had the same idea ;-)

That seems to say that it feels more natural to both of you. I'll update the config.